sdk/mixedreality/Azure.MixedReality.RemoteRendering/src/RemoteRenderingAccount.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System;
+
+namespace Azure.MixedReality.RemoteRendering
+{
+    /// <summary>
+    /// Represents Azure Remote Rendering account details.
+    /// </summary>
+    public class RemoteRenderingAccount
+    {
+        /// <summary>
+        /// Gets the Azure Remote Rendering account domain.
+        /// </summary>
+        public string AccountDomain { get; }
+
+        /// <summary>
+        /// Gets the Azure Remote Rendering account identifier.
+        /// </summary>
+        public Guid AccountId { get; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RemoteRenderingAccount"/> class.
+        /// </summary>
+        /// <param name="accountId">The Azure Remote Rendering account identifier.</param>
+        /// <param name="accountDomain">The Azure Remote Rendering account domain.</param>
```

Comment:
>The Azure Remote Rendering account domain [](start = 40, length = 41)

as a customer i would not know what to fill in here - can we get an example? Same goes for the example README.md